package.json

```diff
@@ -263,6 +261,7 @@
     "lint-staged": "^13.2.3",
     "metro-react-native-babel-preset": "^0.74.1",
     "prettier": "^2.8.3",
+    "react-compiler-runtime": "^19.0.0-beta-63b359f-20241101",
```

Comment:
Instead of the one we had inside `lib`

Reply:
I had done this, but we had to revert due to a crash bug (https://github.com/bluesky-social/social-app/pull/6071)